app/controllers/admin/configs_controller.rb

```diff
@@ -1,39 +1,95 @@
 module Admin
   class ConfigsController < Admin::ApplicationController
-    layout "admin"
+    CAMPAIGN_PARAMS =
```

Comment:
Nice, so much better. If you ever feel like a follow-up PR @joshpuetz, maybe moving these into a file in `app/constants` could be good, similar to `app/constants/site_config.rb`.